### ISSUE TITLE
ERROR: (gcloud.auth.login) unrecognized arguments: --cred-file (did you mean '--flags-file'?) 

### ISSUE THREAD

**Issue by YangJiao0817** [google-github-actions/setup-gcloud#639]
### TL;DR

google-github-actions/setup-gcloud@v1 reported an error ERROR: (gcloud.auth.login) unrecognized arguments: --cred-file (did you mean '--flags-file'?)

### Action YAML

```yaml
      - uses: 'google-github-actions/auth@v1'
        with:
          credentials_json: '${{ secrets.GCP_CREDENTIALS }}'
      - uses: google-github-actions/setup-gcloud@v1
        with:
          version: '285.0.0'
```
```


### Log output

```text
Run google-github-actions/setup-gcloud@v1
  with:
    version: 285.0.0
    skip_install: false
  env:
    DOCKER_COMPOSE_VERSION: 1.23.0
    BUILD_PACKAGE: true
    CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/harbor/harbor/gha-creds-3d9bf86c4a872dbf.json
    GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/harbor/harbor/gha-creds-3d9bf86c4a872dbf.json
    GOOGLE_GHA_CREDS_PATH: /home/runner/work/harbor/harbor/gha-creds-3d9bf86c4a872dbf.json
    CLOUDSDK_CORE_PROJECT: eminent-nation-87317
    CLOUDSDK_PROJECT: eminent-nation-87317
    GCLOUD_PROJECT: eminent-nation-87317
    GCP_PROJECT: eminent-nation-87317
    GOOGLE_CLOUD_PROJECT: eminent-nation-87317
/usr/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/work/_temp/a75d5fee-7821-499e-883c-b8edc55446ce -f /home/runner/work/_temp/a202e60a-86c2-4a31-b6bc-44abf9aa10d3
Error: google-github-actions/setup-gcloud failed with: failed to execute command `gcloud --quiet auth login --force --cred-file /home/runner/work/harbor/harbor/gha-creds-3d9bf86c4a872dbf.json`: ERROR: (gcloud.auth.login) unrecognized arguments: --cred-file (did you mean '--flags-file'?) 
```

**Comment by sethvargo** [comment]
Hi @YangJiao0817 

[gcloud version 285.0.0](https://cloud.google.com/sdk/docs/release-notes#28500_2020-03-17) is over 3 years old. Are you able to use a [more recent version](https://cloud.google.com/sdk/docs/release-notes#43000_2023-05-09) like 430.0.0?

**Comment by YangJiao0817** [comment]
@sethvargo Thanks, this worked.
